### DISCUSSION TITLE
Worker unhealthy

### DISCUSSION THREAD

**Discussion by xfy777** [netbox-community/netbox-docker#1073] [Getting Help]
### Current Behavior

sysop@netbox:~/netbox-docker$ sudo docker-compose up -d
netbox-docker_redis_1 is up-to-date
netbox-docker_postgres_1 is up-to-date
netbox-docker_redis-cache_1 is up-to-date
netbox-docker_netbox_1 is up-to-date

ERROR: for netbox-worker  Container "1b627a3a998f" is unhealthy.

ERROR: for netbox-housekeeping  Container "1b627a3a998f" is unhealthy.

### Expected Behavior

Container comes up lol

### Docker Compose Version

docker-compose version 1.29.2, build unknown

### Docker Version

```text
Client:
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.3
 Git commit:        24.0.5-0ubuntu1~22.04.1
 Built:             Mon Aug 21 19:50:14 2023
 OS/Arch:           linux/amd64
 Context:           default
```

### The git Revision

b47e85ab3f2261021adf99ae9de2e9692fd674c3

### The git Status

```text
main
```

### Startup Command

sudo docker-compose up -d

### NetBox Logs

```text
N/A
```

### Content of docker-compose.override.yml

```yaml
version: '3.4'
services:
  netbox:
    ports:
      - 8000:8080
```

**Comment by NTNTP** [comment]
Hello, 

If you do docker stop with the container id, normaly it should work !
